docs/orchestrating-elastic-stack-applications/apm-server.asciidoc

```diff
@@ -75,6 +75,28 @@ apm-server-quickstart-apm-server-69b447ddc5-fflc6   1/1     Running   0
 [id="{p}-apm-advanced-configuration"]
 == Advanced configuration
 
+[id="{p}-apm-agent-central-configuration"]
+=== Use APM Agent central configuration
+link:https://www.elastic.co/guide/en/kibana/current/agent-configuration.html[APM Agent configuration management] allows you to configure your APM Agents centrally from within the Kibana APM app. In order to use APM Agent central configuration you need to set up a Kibana endpoint in your APM Server configuration. Using a Kibana managed by ECK makes that as simple as adding a `kibanaRef` attribute to the APM Server specification:
```

Comment:
```suggestion
link:https://www.elastic.co/guide/en/kibana/current/agent-configuration.html[APM Agent configuration management] allows you to configure your APM Agents centrally from within the Kibana APM app. In order to use this feature, APM Server needs to be configured with connection details of the Kibana instance. If Kibana is managed by ECK, this can be achieved by simply adding a `kibanaRef` attribute to the APM Server specification:
```